api/descriptions/collections/post.md

```diff
@@ -8,20 +8,30 @@ To invoke this endpoint in the Console UI:
 
 ### cURL Request
 
-Refer to the following example cURL command that creates a new collection named `my-collection` and captures all container images named `ubuntu:18.04`:
+Refer to the following example cURL command that creates a new collection named `my-collection`, specifies a HEX color value of #AD3C21, and captures all container images named `ubuntu:18.04`:
```

Comment:
@ssugandh What happens if you don't specify a color? Does the system randomly assign the collection a unique color? Please test it in your lab setup. If it does, then remove this param from the example curl command.

I want our example curl commands to show only the **minimum params** do required to do something **useful in the real world**. (Sometimes our API lets you specify even fewer params then we show in our curl commands, but the created object doesn't do anything)

Reply:
@ssugandh The reason I want our curl commands specified this way is because our OpenAPI spec doesn't declare which params are optional and which are required. As such, I want our curl commands to act as a stand-in for this missing info.

Customers can then take our "minimally useful" curl command, and build on it to implement their own use case.

Reply:
@iansk : I've verified that only three resources are manadatory to use the collection, name, images, and labels. I have described about it in the description. I've retained the example with some resources just as a reference. Let me know if you still think otherwise, I can remove the non-mandatory resources from the example. 

In my other experiences to validate cURL example for creating users, I struggled to get the exact names of the property "role". I had to create users in the UI and use GET to understand all the exact values of the "role" used in the JSON file.

For color, should we not update about the random color in the parameter description?